server/py/services/api/tests/unit/db/test_artifacts.py

```diff
@@ -1557,12 +1557,16 @@ def test_list_artifact_for_tagging_fallback(self):
     @pytest.mark.parametrize("limit", [None, 6])
     def test_list_artifacts_returns_elements_by_order_updated_field(self, limit):
         project = "artifact_project"
+        artifact_kinds = ArtifactCategories.all()
```

Comment:
isn't this only implemented in https://github.com/mlrun/mlrun/pull/7328 ?

Reply:
yeah, I will wait with other pr to be merged as I worked on both in parallel. so that is a glitch